app/ui-react/packages/ui/package.json

```diff
@@ -46,21 +46,24 @@
     "jest": "^25.1.0",
     "jest-cli": "^25.1.0",
     "microbundle": "^0.11.0",
+    "monaco-editor": "^0.19.3",
+    "monaco-editor-webpack-plugin": "^1.8.2",
     "react": "^16.6.0",
     "react-content-loader": "^4.0.1",
     "react-docgen-typescript-loader": "^3.1.0",
     "react-dom": "16.9.0",
     "react-router": "^5.0.0",
     "react-router-dom": "^5.0.0",
     "rimraf": "^3.0.2",
-    "ts-jest": "^25.2.1",
+    "ts-jest": "^24.0.0",
```

Comment:
Was ts-jest going backwards required for monaco editor? are all of the other added dependencies still required after moving some of the code to api?

Reply:
reverted dependencies and removed patternfly